packages/spawn/spawn.v0.15.0/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+synopsis: "Spawning sub-processes"
+description: """
+Spawn is a small library exposing only one functionality: spawning sub-process.
+
+It has three main goals:
+
+1. provide missing features of Unix.create_process such as providing a
+working directory
+
+2. provide better errors when a system call fails in the
+sub-process. For instance if a command is not found, you get a proper
+[Unix.Unix_error] exception
+
+3. improve performances by using vfork when available. It is often
+claimed that nowadays fork is as fast as vfork, however in practice
+fork takes time proportional to the process memory while vfork is
+constant time. In application using a lot of memory, vfork can be
+thousands of times faster than fork.
+"""
+maintainer: ["Jane Street developers"]
+authors: ["Jane Street Group, LLC"]
+license: "MIT"
+homepage: "https://github.com/janestreet/spawn"
+doc: "https://janestreet.github.io/spawn/"
+bug-reports: "https://github.com/janestreet/spawn/issues"
+depends: [
+  "dune" {>= "2.5"}
```

Comment:
```suggestion
  "dune" {>= "2.8"}
```

Reply:
Sorry, I just copied the metadata from 0.14, which was of course wrong. I now updated the opam file in accordance with the one in the source archive.